sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/AggregatePushDownUtils.scala

```diff
@@ -138,4 +141,19 @@ object AggregatePushDownUtils {
     converter.convert(aggregatesAsRow, columnVectors.toArray)
     new ColumnarBatch(columnVectors.asInstanceOf[Array[ColumnVector]], 1)
   }
+
+  /**
+   * Return the schema for aggregates only (exclude group by columns)
+   */
+  def getSchemaWithoutGroupingExpression(
```

Comment:
nit: maybe `getSchemaWithoutGroupColumns`?

Reply:
Changed to `getSchemaWithoutGroupingExpression` per this suggestion https://github.com/apache/spark/pull/34445#discussion_r740418455